src/client/src/apps/SimpleLauncher/applicationConfigurations.tsx

```diff
@@ -121,6 +121,19 @@ export const appConfigs: ApplicationConfig[] = [
       },
     },
   },
+  {
```

Comment:
I would restrict it to `process.env.NODE_ENV === 'development'` for now

Reply:
done